driverkit/utils/driverstats

```diff
@@ -10,8 +10,14 @@ source "${currentdir}/parseyaml"
 TARGET_VERSION=${TARGET_VERSION:-"*"}
 TARGET_DISTRO=${TARGET_DISTRO:-"*"}
 TARGET_KERNEL=${TARGET_KERNEL:-"*"}
+TARGET_ARCH=${TARGET_ARCH:-"*"}
 
-configs=(config/${TARGET_VERSION}*/${TARGET_DISTRO}_${TARGET_KERNEL}-*.yaml)
+# Only load old configs if x86_64 or * arch is requested
+# TODO: rm this once old format support is dropped
+if [ "${TARGET_ARCH}" = "*" ] || [ "${TARGET_ARCH}" = "x86_64" ]; then
```

Comment:
If users filter for TARGET_ARCH, only load old config if either "*" (any) or "x86_64" is requested.